src/net.h

```diff
@@ -113,6 +113,14 @@ struct CSerializedNetMsg
     std::string command;
 };
 
+enum class ConnectionType {
+    INBOUND, // peer initiated connections
+    OUTBOUND, // full relay connections (blocks, addrs, txns)
```

Comment:
Maybe add that these connections are made automatically to addresses selected from AddrMan.

Reply:
added